frontend/src/components/App/App.jsx

```diff
@@ -24,31 +24,27 @@ export default function App() {
         localStorage.setItem("user", JSON.stringify(user));
     }, [user]);
 
+    const requireAuth = (element) => {
+        return user ? element : <LoginForm />;
```

Comment:
It is better to stick to check for a boolean explicitly: `return user != null ? element : <LoginForm />;`
`user` is an object, not a boolean. By doing `user ?` you're relying in JS coercion and expecting an object to be "thruthy" and a null|undefined to be "falsy". 
That is a little hacky, and the reader has to make more effort reading it. An explicit boolean check is more clear, and recommended.